### DISCUSSION TITLE
Hierarchical organization of GitHub.

### DISCUSSION THREAD

**Discussion by MMariusch** [community/community#137353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,
I’m working on setting up a GitHub account for my university, but I’m having trouble figuring out how to organize it properly. Our goal is to structure it so that each department is something like a nested organization. After some research, I found that GitHub doesn’t support this feature, and the closest solution is using Teams.
I created Teams for our departments and added repositories to them. Later I realized that Teams aren’t visible to outsiders. Is there a way to make them visible, or did I take the wrong approach?

Thank you for your help!

**Comment by mohitpaddhariya** [comment]


**Reply by MMariusch** [reply]
Thank you. I think that "topics" will meet our needs. I have another question. 
I know it's possible to display a list of "Most used topics" in Overview. But is there a way to display all topics instead of only the most popular ones? It would also be helpful to see them next to the list of all repositories in the repositories tab.
